src/components/WearablePreview/WearablePreview.tsx

```diff
@@ -21,6 +21,38 @@ import { config } from '../../config'
 
 const debounce = createDebounce()
 
+const safeEncodeParam = (key: string, value: unknown): string => {
+  if (value === undefined || value === null || value === '') {
+    return ''
+  }
+
+  // Handle boolean flags
+  if (typeof value === 'boolean') {
+    return `${key}=${String(value)}`
```

Comment:
If we do `encodeURIComponent(String(value))` wouldn't we get the same value?

Reply:
you're right... method updated